src/main/kotlin/minesweeper/domain/MineMap.kt

```diff
@@ -0,0 +1,37 @@
+package minesweeper.domain
+
+class MineMap(private val mapArea: Pair<Int, Int>, _mines: Mines) {
```

Comment:
다른 개발자도 예상할 수 있는 기능일까요?

```suggestion
class MineMap(private val height: Int, private val width: Int, _mines: Mines) {
```